src/js/view/controls/controlbar.js

```diff
@@ -71,17 +71,17 @@ function createCastButton(castToggle, localization) {
         return;
     }
 
-    const ariaText = localization.cast;
 
     const castButton = document.createElement('button', 'google-cast-button');
-    ariaLabel(castButton, ariaText);
     castButton.setAttribute('type', 'button');
+    castButton.setAttribute('tabindex', '-1');
```

Comment:
wouldn't this prevent users from tabbing to the cast button ? Why do we want that ?

Reply:
There's a div around the cast button, so we only want the div to be tabbed to?

Reply:
👍